lib/deppack.js

```diff
@@ -453,6 +456,7 @@ const exploreDeps = fileList => {
     if (!isJs(fileList, path) || !mediator.npmIsEnabled) return Promise.resolve(x);
 
     const allDeps = isVendor(path) ? [] : detective(x.compiled);
+    const usesProcess = isVendor(path) ? false : x.compiled.indexOf('process.') !== -1;
```

Comment:
I suspect this would yield false positives (matching `myprocess.whatever()`, which isn't the end of the world, but it also could fail to match something like `getEnv = proc => proc.env.NODE_ENV; getEnv(process);`. Perhaps a bit contrived. I know regexes are more expensive, but something like `/\bprocess\b/` might be better.